src/service/dashboards/mod.rs

```diff
@@ -26,14 +26,14 @@ use config::{
 };
 
 use super::folders;
+pub use crate::service::db::dashboards::ListParams;
```

Comment:
I'm just re-exporting the `ListParams` type here so that it's easier for the HTTP layer to access when calling into the Service layer. Eventually I would like to just get rid of `crate::service::db::dashboards` entirely since it doesn't really do much now that we are using an actual `dashboards` table rather than the `meta` table.